spv/sync.go

```diff
@@ -143,6 +143,11 @@ func (s *Syncer) synced() {
 	}
 }
 
+// GetRemotePeers returns a map of connected remote peers.
+func (s *Syncer) GetRemotePeers() map[string]*p2p.RemotePeer {
+	return s.remotes
```

Comment:
This is not concurrent safe and will cause data races.

Reply:
How best should it be implemented?

Reply:
It's been fixed.